src/domain/repository/servicesQueryRepo.go

```diff
@@ -2,13 +2,16 @@ package repository
 
 import (
 	"github.com/goinfinite/os/src/domain/dto"
-	"github.com/goinfinite/os/src/domain/entity"
-	"github.com/goinfinite/os/src/domain/valueObject"
 )
 
 type ServicesQueryRepo interface {
-	Read() ([]entity.InstalledService, error)
-	ReadByName(name valueObject.ServiceName) (entity.InstalledService, error)
-	ReadWithMetrics() ([]dto.InstalledServiceWithMetrics, error)
-	ReadInstallables() ([]entity.InstallableService, error)
+	ReadInstalledItems(
```

Comment:
Like I mentioned, please check all the code for repeated errors. I will not be marking where it's repeating.